Code/Editor/CryEdit.cpp

```diff
@@ -3728,24 +3728,24 @@ void CCryEditApp::OnToolsPreferences()
 //////////////////////////////////////////////////////////////////////////
 void CCryEditApp::OnSwitchToDefaultCamera()
 {
-    CViewport* vp = GetIEditor()->GetViewManager()->GetSelectedViewport();
-    if (CRenderViewport* rvp = viewport_cast<CRenderViewport*>(vp))
-    {
-        rvp->SetDefaultCamera();
-    }
+    //CViewport* vp = GetIEditor()->GetViewManager()->GetSelectedViewport();
```

Comment:
Please remove (all commented code in this file should be good to go)

Reply:
Addressed in 70b430082f